README.md

```diff
@@ -29,9 +29,9 @@ irc.mozilla.org.
 
 ### Browser Extensions
 
-#### Firefox
+#### Firefox and Seamonkey
 
-PDF.js is built into version 19+ of Firefox, however one extension is still available:
+PDF.js is built into version 19+ of Firefox, however one extension is still available, in Seamonkey it needs to be manually installed:
```

Comment:
> why not Seamonkey?

Good question.. Probably because we don't know if it works there... and lack of testing on Seamonkey.